Cargo.Bazel.toml.lock

```diff
@@ -3220,8 +3219,9 @@ dependencies = [
  "prometheus-parse",
  "proptest",
  "proptest-derive",
- "prost",
- "prost-build",
+ "prost 0.12.2",
+ "prost 0.13.2",
+ "prost-build 0.13.2",
```

Comment:
This seems to duplicate dependencies. Can you check why both `0.12.2` & `0.13.2` are in use?

Reply:
[pprof](https://docs.rs/pprof/latest/pprof/) depends on prost 0.12. It has not been upgraded yet

Reply:
can we upgrade it then? or does this cause failures?

Reply:
The latest version of pprof uses 0.12, so it can't be upgraded now. It should be easy once the new version is released

Reply:
Gotcha. Can we wait for the new pprof version to be released then or is there some urgency in bumping those deps now?

Reply:
Ideally not as this is my final day at DFINITY and it is also a step in the right direction regarding cleaning up dependencies which will eventually allow us to not pull the old hyper at all

Reply:
We can't wait for pprof. They upgrade their crate fairly slow. 

Again we should not worry to much about transitive dependencies of our external crates as long as use external crates from credible organisations.

Reply:
For the foreseeable future there always can be multiple versions due to transitive dependencies. This is less of a problem than keeping our direct dependencies stale.